### DISCUSSION TITLE
FEATURE REQUEST: Upate/Pull all repositories

### DISCUSSION THREAD

**Discussion by ionultd** [community/community#8516] [Pull Requests]
Hi,

A lot of GitHub users have numerous repositories that gets quite tedious to manually update so they often resort to running a scheduled/cron script to do so. Please make this ability part of GitHub Desktop so that all us users have to do is hit one button to start a repositories wide pull and update. Especially useful for new users instead of scouring the internet for the best update script.

Thx

**Comment by Katsute** [comment]
Maybe consider opening an issue here: https://github.com/desktop/desktop
